### PR TITLE
Dabra maeso cat192021

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
MY-021 reworked.

### CODE REVIEW

File: ES/ESmy020.xml [hunk 1]
```diff
@@ -688,6 +690,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          <change who="MK" when="2014-10-21">last edited</change>
          <change who="PL" when="2016-05-10">transformed from mycore to TEI P5</change>
          <change who="DN" when="2020-08-20">continued description</change>
+         <change who="DN" when="2020-08-27">incorporated comments from Eugenia Sokolinski</change>
```
Comment (eu-genia) [inline ES/ESmy020.xml]: one more error I missed last time:
ms_i2 title needs ref not corresp
        title type="complete" **ref**="LIT3949MashafaKrL"

File: ES/ESmy021.xml [hunk 2]
```diff
@@ -40,62 +40,64 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                          እሙስና፤ ውስተ፤ ዘኢይማስን፤ እምኢያዓእምሮ፤ ውስተ፡ አእምሮ፡ አማን፤ ከመ፡ በኑኃ፤ ሌሊትኒ፡ በኵሎ፡ ሰላም፨
                         </incipit>
                       </msItem>
 
                       <msItem xml:id="ms_i1.2">
                       <locus from="3va" to="3vb"/>
-                      <title type="complete" corresp="LIT4711Mastabqwe#ZaMutan"/>
-                      <incipit>
-                      <q xml:lang="gez"><hi rend="rubric">ወካዕበ፡ በእንተ፡ እለ፤</hi> ኖሙ፤ አኀዊነ፤
+                      <title type="complete" corresp="LIT4711Mastabqwe#SupplicationDead"/>
```
Comment (eu-genia) [inline ES/ESmy021.xml:54]: all title elements here and elsewhere in the MS need `@ref` and not `@corresp`

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: ES/ESmy020.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply ES/ESmy020.xml]: ok

File: ES/ESmy021.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply ES/ESmy021.xml:54]: sorry, stupid mistake

**Pull request merged**